spec/pluralrules.html

```diff
@@ -66,22 +142,11 @@
       <emu-alg aoid="ResolvePlural">
         1. Assert: Type(_pluralRules_) is an Object, and _pluralRules_'s internal slot[[initializedPluralRules]] is *true*.
         1. Assert: Type(_x_) is a Number value.
-        1. Let _pluralRuleFunction_ be the value of _pluralRules_.[[pluralRule]].
         1. If the _result_ of isFinite(_x_) is *false*, then
-          1. If _x_ is *NaN*, then
-            1. Let _n_ be an ILD String value indicating the *NaN* value.
-          1. Else
-            1. If _x_ < 0, then
-              1. let _n_ be -*Infinity*.
-            1. Else
-              1. Let _n_ be *Infinity*.
-        1. Else
-          1. If _x_ < 0, then
-            1. Let _n_ be -_x_.
-          1. Else
-            1. Let _n_ be _x_.
-        1. NOTE n might need to be transform ToString() to preserve decimals.
-        1. Let _result_ be _pluralRuleFunction_(_n_).
+          1. Return *"other"*.
+        1. Let _pluralRuleFunction_ be the value of _pluralRules_.[[pluralRule]].
```

Comment:
we should probably think about a better way to deal with this. I was wrongly assuming that we will get a plural rule function depending on the `type` option, but really what we should do here is just describe what to do with the record, and the corresponding locale data to produce the right plural form, maybe by just describing that with all that data and type, an implementation can do the right thing do select the right value.